recipes/cryptopp-pem/all/conanfile.py

```diff
@@ -48,7 +48,7 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires(f"cryptopp/{self.version}", transitive_headers=True, transitive_libs=True)
+        self.requires(f"cryptopp/8.8.0", transitive_headers=True, transitive_libs=True)
```

Comment:
Is it not needed then that the `cryptopp-pem` and `cryptopp` versions match?

Reply:
It might be necessary, but it built without issues locally at least. I'll have to take a closer look.

I currently did not have any better ideas for dealing with the `const CryptoPP::DEFAULT_CHANNEL` unresolved external symbol error, when both `cryptopp` and `cryptopp-pem` already set `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE`.